app/src/main/java/Edu/calvin/cs262/teamc/adopt_activity.java

```diff
@@ -0,0 +1,74 @@
+package Edu.calvin.cs262.teamc;
+
+import android.content.Context;
+import android.graphics.drawable.Drawable;
+import android.support.v4.view.PagerAdapter;
+import android.support.v4.view.ViewPager;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.Toast;
+
+public class adopt_activity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_adopt_activity);
+
+        ViewPager viewPager = (ViewPager) findViewById(R.id.dogimage);
+        ImageAdapter adapter = new ImageAdapter(this);
+        viewPager.setAdapter(adapter);
+
+
```

Comment:
I'm gonna nitpick here and say we don't need this much whitespace here

Reply:
Fixed.